pkg/daemon/agent/flexvolume/attachment/crd.go

```diff
@@ -51,22 +43,7 @@ type crd struct {
 
 // CreateController creates a new controller for volume attachment
 func New(context *clusterd.Context) (Attachment, error) {
-
-	// CRD is available on v1.7.0. TPR became deprecated on v1.7.0
```

Comment:
What's the experience for someone deploying on 1.6 or older? do they get a decent error message or is it somehow obvious what the problem is?

Reply:
I would expect that when you create the rook-operator.yaml, it will immediately complain that it can't create the CRDs.